R/derive_param_clinbenefit.R

```diff
@@ -0,0 +1,266 @@
+#' Derive a Clinical Benefit Parameter
+#'
+#' Add a clinical benefit/disease control parameter to the input dataset.
+#'
+#' @details
+#' Clinical benefit/disease control is first identified for looking for subjects
+#' having response status, and then derived for subjects that have at least one
+#' evaluable non-PD response assessment prior to first PD (Progressive Disease)
+#' (i.e., inclusive of `CR`, `PR`, `SD` and exclusive of `NA`, `NE`, `ND`, and
+#' `PD`) and after a specified amount of time from a reference date (`ref_start_window`).
```

Comment:
This is not completely clear. For example, how is "NON-CR/NON-PD" handled?

Reply:
updated to just say exclusive of na/ne/nd/pd